src/models/user.js

```diff
@@ -90,6 +90,11 @@ export default (sequelize, DataTypes) => {
       onUpdate: 'CASCADE',
       onDelete: 'CASCADE'
     });
+    User.hasMany(models.FavoriteArticle, {
```

Comment:
Doesn't this need a migration?

Reply:
There's a migration file for the model that I created, but as I wanted to add the relation between the user model and the introduced model